variables.tf

```
@@ -0,0 +1,319 @@
+variable "namespace" {
+  type        = "string"
+  description = "Namespace (e.g. `cp` or `cloudposse`)"
+}
+
+variable "stage" {
+  type        = "string"
+  description = "Stage (e.g. `prod`, `dev`, `staging`)"
+}
+
+variable "name" {
+  type        = "string"
+  description = "Application or solution name (e.g. `app`)"
+  default     = "ecs"
+}
+
+variable "delimiter" {
+  type        = "string"
+  default     = "-"
+  description = "Delimiter to be used between `namespace`, `stage`, `name` and `attributes`"
+}
+
+variable "attributes" {
+  type        = "list"
+  default     = []
+  description = "Additional attributes (e.g. `1`)"
+}
+
+variable "tags" {
+  type        = "map"
+  default     = {}
+  description = "Additional tags (e.g. map(`BusinessUnit`,`XYZ`)"
+}
+
+variable "default_backend_image" {
+  type        = "string"
+  default     = "cloudposse/default-backend:0.1.2"
+  description = "ECS default (bootstrap) image"
+}
+
+variable "github_oauth_token" {
+  type        = "string"
+  description = "GitHub Oauth token. If not provided the token is looked up from SSM."
+  default     = ""
+}
+
+variable "github_oauth_token_ssm_name" {
+  type        = "string"
+  description = "SSM param name to lookup GitHub OAuth token if not provided"
+  default     = ""
+}
+
+variable "atlantis_enabled" {
```

```suggestion
variable "enabled" {
```

Use enabled for consistency with other modules